src/tsickle.ts

```diff
@@ -2130,6 +2130,23 @@ export function emitWithTsickle(
   };
 }
 
+function isSymbolDeclaredInSourceFile(symbol: ts.Symbol, sourceFile: ts.SourceFile) {
+  const declarations = symbol.getDeclarations();
+  if (!declarations) {
+    // If there are no declarations, err on the side of safety and emit the clutz
+    // aliases
```

Comment:
This comment is a bit weird because it comes out of nowhere (it looks like the function is just checking where a symbol lives, but now it's talking about clutz).  Maybe a better comment is a function doc that says "if there are no declarations, returns true" without talking about clutz and then documenting the clutz thing at the usage site?  Not sure.

Reply:
Another idea: make this function take the declaration list rather than the symbol itself, and have the caller pre-filter out symbols that have no declaration.

Reply:
Good suggestion, that was bugging me!